cmd/jujud/upgrade_mongo.go

```diff
@@ -230,7 +230,7 @@ func (u *UpgradeMongoCommand) run() (err error) {
 		return errors.Annotate(err, "could not create a temporary directory for the migration")
 	}
 
-	logger.Infof("begin migration to mongo 3")
+	log.Infof("begin migration to mongo 3")
```

Comment:
No need for this change

Reply:
this package was wrongly created one `log` and one `logger`. 
`logger` belongs to `bootstrap cmd file` which has been moved to `./agent` package.
I can rename `log` in this package to `logger`.

Reply:
actually, upgrade_mongo.go is obsolete and can be totally deleted

Reply:
yes, removed